exporter/views.py

```diff
@@ -47,7 +47,12 @@ class ExportCampaignToCSV(TemplateView):
     Exports the most recent transcription for each asset in a campaign
     """
 
-    @method_decorator(login_required)
+    @method_decorator(permission_required("concordia.add_campaign"))
+    @method_decorator(permission_required("concordia.change_campaign"))
```

Comment:
Do we care about these permissions specifically or should we just use `staff_member_required`?

Reply:
I guess it could be simplified to `staff_member_required` - at this point they're all the same but that could change.